Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -218,6 +270,12 @@ theorem TendstoInMeasure.exists_seq_tendsto_ae (hfg : TendstoInMeasure μ f atTo
   exact h_tendsto x
 
 theorem TendstoInMeasure.exists_seq_tendstoInMeasure_atTop {u : Filter ι} [NeBot u]
+    [IsCountablyGenerated u] {f : ι → α → E} {g : α → E} (hfg : TendstoInMeasure μ f u g) :
+    ∃ (ns : ℕ → ι) (_ : Tendsto ns atTop u),  TendstoInMeasure μ (fun n => f (ns n)) atTop g := by
+  obtain ⟨ns, h_tendsto_ns⟩ : ∃ ns : ℕ → ι, Tendsto ns atTop u := exists_seq_tendsto u
+  exact ⟨ns, h_tendsto_ns, fun ε hε => (hfg ε hε).comp h_tendsto_ns⟩
+
+theorem TendstoInMeasure.exists_seq_tendstoInMeasure_atTop' {u : Filter ι} [NeBot u]
```

Comment:
I think this lemma is strictly weaker than the previous. Can you remove it?

Reply:
Yes, I removed it. The weaker version was part of the file before my PR.